src/reducers/globalState.js

```diff
@@ -25,7 +25,7 @@ export default function reducer(_state, action) {
   let state = { ..._state };
   let deviceIndex = null;
   switch (action.type) {
-    case Types.ACTION_STARTUP_DATA:
+    case Types.ACTION_STARTUP_DATA: {
```

Comment:
why do these have brackets and the other cases don't?

Reply:
Fixing this warning "Unexpected lexical declaration in case block" it was only for the blocks that had variables within case scope. Should I just ignore that warning or put brackets on the other cases as well?